src/AppBundle/Resources/views/Pa/ClientProfile/overview.html.twig

```diff
@@ -316,7 +316,7 @@
     } %}
 
     {% include 'AppBundle:Pa/ClientProfile:_reports.html.twig' with {
-    reports: report.client.reports,
+    reports: report.client.getSubmittedReports(),
```

Comment:
report.client.submittedReports would be more in line

Reply:
sure